gcputil/resource_name.go

```diff
@@ -11,7 +11,7 @@ const (
 	resourceIdRegex   = "^[^\t\n\f\r]+$"
 	collectionIdRegex = "^[a-z][a-zA-Z]*$"
 
-	fullResourceNameRegex = "^//([a-z]+).googleapis.com/(.+)$"
+	fullResourceNameRegex = "^//([a-z]+)\\.googleapis\\.com/(.+)$"
```

Comment:
May want to use backticks instead of double quotes so we don't have to double-escape things?

Reply:
Good call, thanks!